debugger/src/debug_adapter/dap_types.rs

```diff
@@ -7,7 +7,7 @@ use schemafy::schemafy;
 use serde::{Deserialize, Serialize};
 use std::convert::TryFrom;
 
-schemafy!(root: debugserver_types "src/debugProtocol.json");
+schemafy!(root: debugserver_types "src/debug_adapter/debugProtocol.json");
```

Comment:
```suggestion
schemafy!(root: debugserver_types "src/debug_adapter/schema.json");
```

I think if we call this schema it is more obvious that this is given :)

Reply:
Done. Also added a comment to explain our decisions.